app/models/rbac.rb

```diff
@@ -222,7 +222,7 @@ def self.find_targets_filtered_by_parent_ids(parent_class, scope, find_options,
   end
 
   def self.find_targets_filtered_by_ids(scope, find_options, u_filtered_ids, filtered_ids)
-    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count
+    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count(:all)
```

Comment:
Thanks for the clarification and review @NickLaMuro @chrisarcand.  After this discussion I feel much safer with this change than ALL THAT AREL 👇 ... I'll continue reviewing the arel in the morning.